django/thunderstore/frontend/templatetags/qurl.py

```diff
@@ -14,14 +14,15 @@ def __init__(self, param_key, param_val, removals: List[str]):
 
     def render(self, context):
         request = context["request"]
+        community_site = context["community_site"]
```

Comment:
This will not always be available and can't be relied upon. Was there some reason the `request.path` wasn't usable?

Reply:
Ah, yes we actually should use `request.path` here, because otherwise the ordering links would always go to the community packages view, rather than just ordering for example the owner specific packages.

Nice catch, I'll fix this 👌